ui.py

```diff
@@ -7,7 +7,7 @@ def display_menu_get_choice(menu):
      :returns: the user's choice """
     while True:
         print(menu)
-        choice = input('Enter choice? ')
+        choice = input('Enter choice? ').capitalize()
```

Comment:
Is capitalize the best choice? Works for q and Q but what if the option was 'QUIT' and you wanted to allow 'quit'?

Reply:
Hi Clara, 
thank for your feedback.
I switched to Lower and created a variable for quit key in main.
please let me know if you were trying to recommend something else.

Thanks

Reply:
Perhaps it would be better if we didn't have a Quit_key variable in main, and rather change the if statement to

if choice == 'q':
    break

As well as change line 30 to be 

menu.add_option('q', 'Quit', quit_program)

I think it would be a little better to not introduce new variables if we don't have to. 

A potential problem I see is that now when the menu is printed, it shows the user a lowercase 'q' which I think looks a little odd. 

Let me know what you think!

Reply:
Also! What do you think about leaving the quit message as the same as well? It was originally 'Thanks for using the book reading list - happy reading!' and was changed to 'Thanks and bye!' I think the original is a little more verbose but a little more user friendly. Let me know what you think!

Reply:
Disregard my last comment, I was confused about what came from main and what didn't. I'll have to be better about checking what each commit did!